UnitsNet/GeneratedCode/Quantities/AmountOfSubstance.NetFramework.g.cs

```diff
@@ -877,7 +877,7 @@ public static AmountOfSubstanceUnit GetUnitFor(BaseUnits baseUnits)
             if(baseUnits == null)
                 throw new ArgumentNullException(nameof(baseUnits));
 
-            var unit = Info.UnitInfos.Where((unitInfo) => unitInfo.BaseUnits.ExistsIn(baseUnits)).FirstOrDefault();
+            var unit = Info.UnitInfos.Where((unitInfo) => unitInfo.BaseUnits.IsSubsetOf(baseUnits)).FirstOrDefault();
             if(unit == null)
                 throw new NotImplementedException($"No LengthUnit was found for the given BaseUnits.");
```

Comment:
Substitute `LengthUnit`

Reply:
I moved this to QuantityInfo anyways. Easier when dealing with interfaces or statics (for later)